.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  php-tests:
+    name: Run PHP Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['5.6', '7.4']
```

Comment:
5.6 is a special case, so this should go in the matrices `include` key since we don't want to generate a permutation of each WP version for it.

This would only be `[ '7.4' ]`.